axes/migrations/0009_accesslog_autologout_accesslog_user.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.7 on 2023-11-16 11:42
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("axes", "0008_accessfailurelog"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="accesslog",
+            name="autologout",
+            field=models.BooleanField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="accesslog",
+            name="user",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
```

Comment:
It looks like this column may have been missed previously and is getting picked up now. I've confirmed the column exists already so it won't be reflected in the sql migrations in the tool repo

Reply:
Thanks! Yeah, goes to show how much we really don't use migrations :sweat_smile: